L/librdkafka/build_tarballs.jl

```diff
@@ -35,7 +33,7 @@ make install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = expand_cxxstring_abis(supported_platforms())
+platforms = expand_cxxstring_abis(supported_platforms(exclude = x -> Sys.isbsd(x) && !Sys.isapple(x)))
```

Comment:
Do you want to exclude FreeBSD mainly?

```suggestion
platforms = expand_cxxstring_abis(supported_platforms(exclude = Sys.isfreebsd)
```